packages/opticss/src/initializers/initKnownIdents.ts

```diff
@@ -31,24 +28,4 @@ export function initKnownIdents(
   pass.identGenerators.reserve("class", ...opts.omitIdents.class);
   pass.identGenerators.reserve("id", ...opts.omitIdents.id);
 
-  // Reserve every existing identifier used in any of the files
```

Comment:
Largest change – I don't see the value in automagically reserving selectors that we are going to re-name later. It just causes magic holes in our naming schemes and makes the optimized output a little harder to reason about.

Oh, as I write this I understand. This is for when selector rewriting is turned off and we're generating new selectors.

I'm going to separate these concerns and make the tests a bit more robust. If selector rewriting is turned off, we want to make sure we don't duplicate selectors. If selector rewriting is turned on, there is no need to reserve discovered selectors.

Reply:
The other reason I do this is because if the author writes selectors in another css file that isn't optimized, I don't want that selector to accidently match a different element. I think it's a good idea to keep those name spaces distinct. Holes in our namespace isn't a big deal IMO.